src/queryResult/queryResultWebViewController.ts

```diff
@@ -162,6 +169,90 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
             }
             await this.createPanelController(message.uri);
         });
+        this.registerReducer("getExecutionPlan", async (state, payload) => {
```

Comment:
Is this a merge error? IIRC these reducer/handler registrations have been moved to utils.ts